src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.html

```diff
@@ -0,0 +1,68 @@
+<div class="link-container">
+    <div class="link-header" (click)="toggleLinkComponent()">
+        <span class="content__title pull-left">
+            <!-- Refactoring needed -->
+            <i *ngIf="showLinkComponent" class="fa fa-angle-down"></i>
+            <i *ngIf="!showLinkComponent" class="fa fa-angle-up"></i>
+        </span>
+        <div class="link-header-nav"><span class="font18">{{totalLinks}}</span><span class="link-header-nav-text">Total Links</span></div>
+        <div class="link-header-nav" *ngFor="let linkType of linkTypes">
+            <span class="font18">{{linksGroupCount[linkType['id']]}}</span><span class="link-header-nav-text">{{linkType.attributes['name']}}</span>
+        </div>
+    </div>
+    <div class="link-content" *ngIf="showLinkComponent">
+        <!-- Refactoring needed to toogle view -->
+        <div *ngIf="totalLinks == 0 && !showLinkView" class="width100 pull-left paddingT20 paddingB20 link-content-create-link">
+            <i class="pficon pficon-add-circle-o add-circle-icon dib"></i>
+            <h2 class="width100 dib">Work Item Links</h2>
+            <p class="width100 dib">There are no items linked to this issue</p>
+            <button class="btn btn-primary dib create-link-btn" (click)="toggleLinkView()">Create Link</button>
+        </div>
+        <!-- Refactoring needed to toogle view -->
+        <div *ngIf="totalLinks >0 || showLinkView" class="width100 pull-left">
+            <div>
+                <div class="width100 pull-left link-item-hdr" (click)="toggleLinkCreator()">
+                    <p class="pull-left">This item, {{ workItem.fields['system.title'] }}</p>
+                    <i class="pficon pficon-close pull-right font16"></i>
+                </div>
+                <div  *ngIf="showLinkCreator" class="width100 pull-left padding20">
+                    <div class="width100 pull-left">
+                        <div class="col-md-5 col-sm-6 col-xs-6">
+                            <select [ngModel]="selectedLinkType" (ngModelChange)="onSelectRelation($event)" class="width100 pull-left">
```

Comment:
We never use <select> <option> because it pulls the default design of OS. Try using <ul> <li> following the dropdown menu we have.